pkg/pfcpsim/ies.go

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Open Networking Foundation
+
+package pfcpsim
+
+import (
+	"github.com/wmnsk/go-pfcp/ie"
+	"net"
+)
+
+type IEMethod uint8
+
+const (
+	Create IEMethod = iota
+	Update
+	Delete
+)
+
+const (
+	dummyPrecedence = 100
+)
+
+var uplinkPDR = ie.NewCreatePDR(
+	ie.NewPDRID(1),
+	ie.NewPrecedence(dummyPrecedence),
+	ie.NewPDI(
+		ie.NewSourceInterface(ie.SrcInterfaceAccess),
+		ie.NewFTEID(0x01, 0x30000000, net.ParseIP("198.18.0.1"), nil, 0),
+		ie.NewUEIPAddress(0x2, "16.0.0.1", "", 0, 0),
+		ie.NewSDFFilter("permit out ip from any to assigned", "", "", "", 1),
+	),
+	ie.NewOuterHeaderRemoval(0, 0),
+	ie.NewFARID(1),
+	ie.NewQERID(1),
+	ie.NewQERID(4),
+)
+
+// TODO: use builder pattern to create PDR IE
```

Comment:
I plan to implement builder pattern on a subsequent PR, if you agree. I'd like to avoid making this PR too big

Reply:
yes, definitely